source/_integrations/google_translate.markdown

```diff
@@ -45,4 +69,73 @@ A full configuration sample including optional variables:
 tts:
   - platform: google_translate
     language: "de"
+    tld: com
+```
+
+## Service say
+
+The `google_translate_say` service support `language` and also `options` for set, `tld`. The text for speech is set with `message`. Since release 0.92, service name can be defined in configuration `service_name` option.
```

Comment:
```suggestion
The `google_translate_say` service supports `language` and also `options` for setting `tld`. The text for speech is set with `message`. Since release 0.92, the service name can be defined in the configuration `service_name` option.
```